data-service-client/build.gradle

```diff
@@ -1,6 +1,8 @@
-description 'Blox DataService Client'
+plugins {
+    id "java"
```

Comment:
Ah, looks like this is a [new incubating feature](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block).